bccsp/sw/fileks.go

```diff
@@ -411,20 +407,24 @@ func (ks *fileBasedKeyStore) createKeyStoreIfNotExists() error {
 func (ks *fileBasedKeyStore) createKeyStore() error {
 	// Create keystore directory root if it doesn't exist yet
 	ksPath := ks.path
-	logger.Debugf("Creating KeyStore at [%s]...", ksPath)
+	logger.Debugf("Creating KeyStore at [%s]", ksPath)
 
-	os.MkdirAll(ksPath, 0755)
+	err := os.MkdirAll(ksPath, 0755)
+	if err == nil {
+		logger.Debugf("KeyStore created at [%s]", ksPath)
```

Comment:
If statement should just include the error flow - `if error != nil`

The normal path would then continue, in this case, to debug success and return nil.

See the idiomatic go guidlines, e.g.
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
ok. The statement changed to `if err := func(); err != nil`